database_dfj.sql

```diff
@@ -30,9 +30,27 @@ CREATE TABLE `dfj_location`.`reservation` (
   `bike_id` INT NOT NULL,
   `number_bike` INT NOT NULL,
   `withdrawal_date` DATE NOT NULL,
-  `duration` VARCHAR(100),
+  `duration_id` INT,
   `comment` TEXT,
   `is_validated` VARCHAR(10) DEFAULT NULL,
   PRIMARY KEY (`id`),
   CONSTRAINT FK_BikeReservation FOREIGN KEY (bike_id)
-  REFERENCES bike(id));
+  REFERENCES bike(id)
+  CONSTRAINT FK_DurationReservation FOREIGN KEY (duration_id)
+  REFERENCES duration(id));
+
+CREATE TABLE `dfj_location`.`duration` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(100) NOT NULL,
+  PRIMARY KEY (`id`));
+
+CREATE TABLE `dfj_location`.`prices` (
```

Comment:
si je comprend bien ce que tu veux faire, tu veux une table avec les durées possible, et pour chaque vélo, indiquer le prix pour chaque durée ? si c'est bien ça, c'est une many to many en price et bike, et prices est ta table intermédiaire.
donc dans price tu dois avoir un bike_id et un duration_id, et non un duration_id dans bike